stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/home/HomeViewModel.kt

```diff
@@ -95,7 +95,7 @@ class HomeViewModel(
         when (action) {
             is UiAction.LogoutClicked -> {
                 viewModelScope.launch {
-                    chatClient.disconnect(false).await()
+                    chatClient.disconnect(flushPersistence = true).await()
```

Comment:
Clear the persisted user data when logging out to prevent receiving push-notifications from logged-out users.